raster-foundry/raster-foundry
# Add graphite reporting to metrics

## Overview

This PR points metrics to a graphite server instead of to stdout, since stdout was going to get pretty noisy.

### Checklist


### Notes

Choice between graphite and ganglia was based on ease of setup -- ganglia deps were mad at me, so I switched to graphite and everything was fine. They both have ugly dashboards we can access and docker containers for easy setup, so :man_shrugging:

## Testing Instructions

 * docker-compose up backsplash
 * navigate to `localhost:8090`
 * check out your sweet dashboard
 * wait a minute (don't make any requests since we're segfaulting like mad right now) and then explore down the `com` / `rasterfoundry` / `backsplash` tree of the metrics browser
